pkg/kubectl/resource_printer.go

```diff
@@ -334,9 +358,10 @@ type PrintOptions struct {
 // will only be printed if the object type changes. This makes it useful for printing items
 // received from watches.
 type HumanReadablePrinter struct {
-	handlerMap map[reflect.Type]*handlerEntry
-	options    PrintOptions
-	lastType   reflect.Type
+	handlerMap    map[reflect.Type]*handlerEntry
+	options       PrintOptions
+	lastType      reflect.Type
+	hiddenPodsNum int
```

Comment:
We need to do something similar for replicasets in #27510 and we might need it again for other objects. Can you change the field name to something more generic like `hiddenEntryNum`, `hiddenObjNum`, `hiddenCount` or similar? That way, we can reuse the code and logic.